core/src/surplus.ts

```diff
@@ -152,28 +153,29 @@ export const collectSurplusAuction = async function (network: string, auctionInd
 export const enrichSurplusAuction = async (
     network: string,
     auction: SurplusAuctionActive
-): Promise<SurplusAuctionWithMinimumBid> => {
+): Promise<SurplusAuctionEnriched> => {
+    const unitPrice = auction.receiveAmountDAI.div(auction.bidAmountMKR);
+    const marketUnitPrice = (await convertMkrToDai(network, auction.bidAmountMKR)).div(auction.bidAmountMKR);
```

Comment:
While `unitPrice` = `receiveAmountDAI` / `bidAmountMKR` – is effectively MKR price
The other number is something else: `convertMkrToDai` returns amount of DAI will be paid for `bidAmountMKR`, so in order to get MKR unit price, you need to divide it by the amount of MKR, not DAI.

But in general, we want the opposite number (MKR/DAI):
```suggestion
    const unitPrice = auction.bidAmountMKR.div(auction.receiveAmountDAI);
    const mkrMarketPrice = (await convertMkrToDai(network, auction.bidAmountMKR)).div(auction.bidAmountMKR);
    const marketUnitPrice = new BigNumber(1).div(mkrMarketPrice);
```